test/e2e/page-objects/pages/confirmations/redesign/confirmation.ts

```diff
@@ -93,6 +107,10 @@ class Confirmation {
     );
   }
 
+  async clickFooterCancelButtonAndWaitToDisappear() {
+    await this.driver.clickElementAndWaitToDisappear(this.footerCancelButton);
```

Comment:
nit: I think here, we want `clickElementAndWaitForWindowToClose`, as the dialog closes once the button is clicked. Just a nit, as the above could also serve as flakiness mitigation, but we rather use this method when we are dealing with winow management flakiness, and the other one, when we are remain at the same window, if that makes sense

Reply:
I see, you can dismiss my suggestion :pray: I see what's happening, so this method is used both by the dialog view and for the full view, so in the full view, the window will never be closed. That's why the spec fails.
Thanks for looking into it :heart: